src/domains/tasks/ViewListings/components/ListingsPage.tsx

```diff
@@ -58,10 +58,18 @@ export function ListingsPage() {
       }
     >
       {data ? null : (
-        <>
-          Please <Link href="/login">login</Link> or{" "}
-          <Link href="/signup">signup</Link> to view listings.
-        </>
+        <div
+          style={{
+            marginTop: "1.25em",
+            fontSize: "1.3em",
+            wordWrap: "break-word",
+            wordBreak: "break-word",
+            overflowWrap: "break-word",
```

Comment:
This doesn't seem to be doing anything.

Can you please remove these three lines?